tests/integration/ha_tests/test_ha.py

```diff
@@ -559,28 +559,35 @@ async def test_network_cut(ops_test: OpsTest, continuous_writes, chaos_mesh):
 
     # Wait until Mongodb actually detects isolated instance
     logger.info(f"Waiting until Mongodb detects primary instance {primary.name} is not reachable")
+
     await wait_until_unit_in_status(ops_test, primary, active_unit, "(not reachable/healthy)")
 
     # verify new writes are continuing by counting the number of writes before and after a 5 second
     # wait
     logger.info("Validating writes are continuing to DB")
-    with await get_mongo_client(ops_test, excluded=[primary.name]) as client:
+    with await get_mongo_client(
+        ops_test, excluded=[primary.name], use_subprocess_to_get_password=True
+    ) as client:
         writes = client[TEST_DB][TEST_COLLECTION].count_documents({})
         time.sleep(5)
         more_writes = client[TEST_DB][TEST_COLLECTION].count_documents({})
     assert more_writes > writes, "writes not continuing to DB"
 
     # verify that a new primary got elected, old primary is still cut off
-    new_primary = await get_replica_set_primary(ops_test, excluded=[primary.name])
+    new_primary = await get_replica_set_primary(
+        ops_test, excluded=[primary.name], use_subprocess_to_get_password=True
+    )
     assert new_primary.name != primary.name
 
     # Remove networkchaos policy isolating instance from cluster
     remove_instance_isolation(ops_test)
 
+    time.sleep(60)
```

Comment:
might be nice to add a comment so we know what the sleep is for

Reply:
Done, thank you